File: go-sdk/pkg/models/operations/getinfo.go
package operations

import (
	"github.com/speakeasy-sdks/formance-sdks/go-client-sdk/pkg/models/shared"
)

type GetInfoResponse struct {
	ConfigInfoResponse *shared.ConfigInfoResponse
	ContentType        string
	ErrorResponse      *shared.ErrorResponse
	StatusCode         int64
}

File: ts-sdk/src/sdk/models/operations/getinfo.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import * as shared from "../shared";


export class GetInfoResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  configInfoResponse?: shared.ConfigInfoResponse;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  errorResponse?: shared.ErrorResponse;

  @SpeakeasyMetadata()
  statusCode: number;
}